src/client/components/additionalConfigs/settings/settings.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {settingsIcon} from '../../../assets/icons/icons';
+import './settings.css';
+
+const Settings = () => {
+  return (
```

Comment:
Here return statement can be removed too

Reply:
sorry, my bad, missed first time
done)